closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -314,6 +312,22 @@ int encrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
         return 0;
     }
 
+    /*
+     * This method is for CipherCore to insert the remainder of its buffer
+     * into the ibuffer before a encryptFinal(ByteBuffer, ByteBuffer) operation
```

Comment:
Please correct the comment: it works with `ibuffer_enc`.

Reply:
Done.